java-service.gitlab-ci.yml

```diff
@@ -67,51 +67,28 @@ push_bundle:
     - npm ci
     - 'grunt bundle --packageVersion=$PACKAGE_VERSION --packageName=$CI_PROJECT_NAME'
 
-deploy_cit:
+.deploy_template: &deploy_definiton
```

Comment:
I mainly wanted this in here so I could use this anchor in my ci file, but it appears that the gitlab "include" does not pull in anchors